projects/storefrontapp-e2e-cypress/cypress/integration/b2b/regression/bulk-pricing/b2b-bulk-pricing.e2e-spec.ts

```diff
@@ -0,0 +1,24 @@
+import * as bulkPricingActions from '../../../../helpers/b2b/b2b-bulk-pricing';
+
+describe('B2B - Bulk Pricing', () => {
```

Comment:
Can we also enable mobile test as was demoed on last sprint demo?

Reply:
done